### PR TITLE
-- adding /movie/{id}/tranlations endpoint

### PR DESCRIPTION
**Pull request by ssouris**
Sorry for the * imports Intellij did that automatically.

### CODE REVIEW

File: src/test/java/com/uwetrottmann/tmdb/services/MoviesServiceTest.java [hunk 1]
```diff
@@ -199,6 +201,16 @@ public void test_videos() {
     }
 
     @Test
+    public void test_translations() {
+        Translations translations = getManager().moviesService().translations(TestData.MOVIE_ID, null);
+        assertThat(translations).isNotNull();
+        assertThat(translations.id).isEqualTo(TestData.MOVIE_ID);
+        assertThat(translations.translations.get(0).name).isNotNull();
```
Comment (UweTrottmann) [inline src/test/java/com/uwetrottmann/tmdb/services/MoviesServiceTest.java]: Probably better to for each loop through all `Translation` objects to make sure all of them are valid.

**Comment by UweTrottmann** [comment]
No worries. One minor thing to improve with the test, otherwise works and looks good to me!

**Comment by ssouris** [comment]
Oh just committed your suggestion. Thanks

**Comment by UweTrottmann** [comment]
Great, thanks again for your PR!

**Pull request merged**